config/symbols.stmad.txt

```diff
@@ -67,12 +67,10 @@ EntityIntenseExplosion = 0x801964E4;
 EntitySoulStealOrb = 0x801982BC;
 EntityEnemyBlood = 0x80198650;
 EntityRoomForeground = 0x80198B00;
-GetSideToPlayer = 0x801920AC;
 UNK_Invincibility0 = 0x801806B4;
 g_ItemIconSlots = 0x80199E54;
 g_InitializeEntityData0 = 0x80180508;
 g_InitializeData0 = 0x801804FC;
-g_UnkPrimHelperRot = 0x8018D8C0;
```

Comment:
I'm not sure why the linter grabbed these as they're unrelated to this change.

Reply:
Somehow the linter in one of the merged PRs did not pick up there were symbols to be cleaned-up. It is an issue in the CI linter step.